apps/emacs/emacs_commands.csv

```diff
@@ -0,0 +1,171 @@
+Command, Key binding, Short form, Spoken form
```

Comment:
Do we want to standardize on a space after comma or not? It's inconsistent.

Reply:
What do you mean by standardize? Do you mean, have spaces in other csv headers? We could do that, I guess, but I think it would be better done in another PR. Or do you mean spaces in the entries of the CSV - this CSV, or all of them?

Reply:
Good point. I just meant within this csv but cross-knausj makes sense too

Reply:
ok, I've consistently added a space after every comma I think.

Reply:
except for double commas, `foo,, bar`; emptry entries don't get a space

Reply:
OK, that works.